test/validationFunctions.test.js

```diff
@@ -0,0 +1,69 @@
+
+const request =  require('supertest');
+const validate = require('../src/validationFunctions');
+
+const isInvalidString = validate.isInvalidString; 
+const isInvalidNum = validate.isInvalidNum; 
+const generateErr400 = validate.generateErr400; 
+const generateErr404 = validate.generateErr404; 
+const checkObjFormat = validate.checkObjFormat; 
+const checkObjKeys = validate.checkObjKeys; 
+const checkObjValues = validate.checkObjValues; 
+ 
+
+describe( "checks for invalid data type", () => {
```

Comment:
I left a couple of comments on #11 which could be addressed here. Once that's done feel free to merge! 🎉